asteroid/dsp/overlap_add.py

```diff
@@ -154,13 +154,12 @@ def _reorder_sources(
     previous = previous.reshape(-1, n_src, frames)
 
     overlap_f = window_size - hop_size
-    pw_losses = PITLossWrapper.get_pw_losses(
-        lambda x, y: torch.sum((x.unsqueeze(1) * y.unsqueeze(2))),
-        current[..., :overlap_f],
-        previous[..., -overlap_f:],
+    pit = PITLossWrapper(
+        lambda x, y: torch.sum(
+            (x[..., :overlap_f].unsqueeze(1) - y[..., -overlap_f:].unsqueeze(2)).abs(), dim=-1
```

Comment:
Here we directly compute the pairwise matrix of absolute difference. We could do something else (squared difference, or correlation) if we want though.

Reply:
Should be solved in #271